Romo/src/Drive/Telepresence/RMTelepresencePresence.m

```diff
@@ -99,11 +99,11 @@ - (void)connect
     }
     
     if (!self.number) {
-        [self fetchNumber:^(NSError *error) {
-            if (!error) {
-                [self connect];
-            }
-        }];
+//        [self fetchNumber:^(NSError *error) {
```

Comment:
Why was it removed?

Reply:
These are part of OpenTok TelepresencePresence.